tools/deployment/sparkClusterInVnetTemplate.json

```diff
@@ -98,7 +98,7 @@
    "id": "[resourceId('Microsoft.Network/virtualNetworks', concat(parameters('clusterName'),'-vnet'))]",
    "subnet": "[concat(resourceId('Microsoft.Network/virtualNetworks', concat(parameters('clusterName'),'-vnet')), concat('/subnets/', parameters('virtualNetworkSubnetName')))]"
   },
-  "scriptActionUri": "https://mmlspark.azureedge.net/buildartifacts/0.5/install-mmlspark.sh"
+  "scriptActionUri": "https://mmlspark.azureedge.net/buildartifacts/0.6/install-mmlspark.sh"
```

Comment:
Also over here: this should not rely on a specific build artifact.  Maybe the template file itself should be something that gets generated to not depend on a specific version like this?